### DISCUSSION TITLE
riscv-gcc-prebuilt: add builded commit hash to doc

### DISCUSSION THREAD

**Discussion by andkae** [stnolting/neorv32#798] [Ideas]
Hi Stephan,

your [Prebuilt GCC](https://github.com/stnolting/riscv-gcc-prebuilt) is a real big deal. It makes the compile of source code easily. But whenever you are in need to build your GCC from scratch it would be very helpful to store also the used commit for build in your table:

![image](https://github.com/stnolting/neorv32/assets/106740887/4d4f118a-8628-4c84-8bac-536698879652)

This table could for instance modified to:

| Status | Release (tag) | Download | GCC | binutils | march | mabi | c-lib |
|:------:|:-------------:|:--------:|:---:|:--------:|:-----:|:----:|:-----:|
| :red_circle: | [rv32i-4.0.0](https://github.com/stnolting/riscv-gcc-prebuilt/releases/tag/rv32i-4.0.0) | [:floppy_disk: download](https://github.com/stnolting/riscv-gcc-prebuilt/releases/download/rv32i-4.0.0/riscv32-unknown-elf.gcc-12.1.0.tar.gz) | [12.1.0](https://github.com/riscv-collab/riscv-gnu-toolchain/commit/a8a98fa6ef410f63b7ef9a5cc3e272f519f13e07) | 2.39 | `rv32i` | `ilp32` | newlib |

Could this suitable?

Thanks and BR,
Andreas

**Comment by stnolting** [comment]
Hey @andkae!

This is a pretty good idea! Could you move this as issue to [riscv-gcc-prebuilt](https://github.com/stnolting/riscv-gcc-prebuilt) (so I do not forget about this the next time I upload a toolchain 😅)?

**Reply by andkae** [reply]
Done in https://github.com/stnolting/riscv-gcc-prebuilt/issues/7
